integrations/apk_analyzers/Dockerfile

```diff
@@ -42,4 +42,3 @@ RUN pip3 install -r requirements.txt --no-cache-dir \
 # Serve Flask application using gunicorn
 EXPOSE 4004
 ENTRYPOINT ["./entrypoint.sh"]
-USER ${USER}
```

Comment:
why is this change part of this PR

Reply:
@mlodic asked to modify the docker-file of `apk_analyzers`, so it can run correctly. It is made in this PR because we are touching docker files anyway.